site/sfguides/src/native-app-chairlift/native-app-chairlift.md

```diff
@@ -0,0 +1,641 @@
+author: Gilberto Hernandez
+id: native-app-chairlift
+summary: This Snowflake Native App demonstrates how a chairlift manufacturer can build a native app to provide chairlift customers with analytics on purchased chairlifts by using sensor data collected from the chairlifts owned by the customer.
+categories: Getting-Started
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, Apps 
+
+# Build a Snowflake Native App to Analyze Chairlift Sensor Data
+<!-- ------------------------ -->
+## Introduction
+Duration: 1
+
+In this Quickstart, you'll build a Snowflake Native Application that can analyze sensor data from chairlifts at different locations at a ski resort. Let's explore the scenario from the perspective of the application provider and an application consumer.
+
+– **Provider** – The provider of the app is a chairlift manufacturer. The app, once installed by a consumer, can run against chairlift sensor data housed in the consumer's account, and provide the consumer with important analytics and insights on the condition of each chairlift. 
+
+– **Consumer** – The consumer of the app is a customer of the chairlift manufacturer. They have purchased and installed chairlifts in various locations across a ski resort they own, and they collect raw sensor data from the chairlifts using a connector that ingests this data directly into their account. The provider's app runs against this data and helps the customer identify chairlifts in need of maintenance.
+
+Within the app, you'll create the following:
+
+- **Dashboard** – A dashboard with a list of sensor warnings from all chairlifts at the ski resort. You'll be able to filter by chairlift and by sensor type.
+
+- **Configuration** – A tab that allows an application admin to toggle sensor warnings about chairlifts.
+
+- **Sensor data** – Visualizations of raw sensor data across all chairlifts at the ski resorts.  You'll be able to filter by chairlift and by sensor type.
+
+Let's get started!
+
+![Sensor data](assets/sensor-data.png)
+
+
+### What You’ll Learn 
+
+- How to create an application package
+- How to create grants in the application package to control access to different parts of the app, depending on Snowflake role 
+- How to create a new version of the app
+- How to install and run the app in a Snowflake account
```

Comment:
Perhaps "How to install and run the app as a different role in the same Snowflake account"? Maybe I'm being too pedantic.

Reply:
I think that's a good callout. I went with:

> How to install and run the app in a Snowflake account using multiple roles